tests/meson.build

```diff
@@ -348,6 +353,9 @@ if get_option('install-test')
   if protobuf_support_is_available
     install_subdir('nnstreamer_protobuf', install_dir: unittest_install_dir)
   endif
+  if mxnet_support_is_available
+    install_subdir('unittest_filter_mxnet', install_dir: unittest_tests_install_dir)
```

Comment:
please check install directory: it would be unittest_install_dir, not unittest_tests_install_dir.